website/data/docs-nav-data.json

```
@@ -743,6 +743,23 @@
       }
     ]
   },
+  {
+    "title": "AWS Lambda <sup>BETA</sup>",
+    "routes": [
+      {
+        "title": "Overview",
+        "path": "lambda"
+      },
+      {
+        "title": "Registration",
+        "path": "lambda/registration"
+      },
+      {
+        "title": "Invocation",
+        "path": "lambda/invocation"
```

```suggestion
        "title": "Register Lambda Functions",
        "path": "lambda/registration"
      },
      {
        "title": "Invoke Lambda Functions",
        "path": "lambda/invocation"
```
We should match the navigation label with the page title.